Dockerfile

```diff
@@ -22,6 +22,10 @@ RUN apt-get update \
     && apt-get install -y --no-install-recommends postgresql-client-$PG_MAJOR \
     && rm -rf /var/lib/apt/lists/*
 
+# Node
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get install -y nodejs && rm -rf /var/lib/apt/lists/*
```

Comment:
This should be in a single RUN line, as the curl|bash will run `apt-get update`, and so the rm has no effect on the previous layer